suites/squid/rbd/tier-3_rbd_migration.yaml

```diff
@@ -90,3 +91,49 @@ tests:
         fio:
           size: 100M
       polarion-id: CEPH-83584070
+
+  - test:
+      desc: Live migration of RAW format images with encryption
+      config:
+        encryption_type:
+          - luks1
+          - luks2
+        rep_pool_config:
+          num_pools: 1
+          do_not_create_image: true
+        ec_pool_config:
+          num_pools: 1
+          do_not_create_image: true
+        fio:
+          size: 10M
+          fs: ext4
+          io: true
+        testconfig:
+          id: CEPH-83596444
+          name: Live migration of raw format images with encryption
+      module: test_rbd_live_migrate_encryption.py
+      name: Live migration of RAW format images with encryption
+      polarion-id: CEPH-83596444
+
+  - test:
+      desc: Live migration of qcow2 format images with encryption
+      config:
+        encryption_type:
+          - luks1
+          - luks2
+        rep_pool_config:
+          num_pools: 1
+          do_not_create_image: true
+        ec_pool_config:
+          num_pools: 1
+          do_not_create_image: true
+        fio:
+          size: 10M
+          fs: ext4
+          io: true
+        testconfig:
```

Comment:
We may not need `testconfig` since we can use `desc` and `polarionId` to represent the same attributes when dealing with them in test modules. May I know if there is any reason for having this here?